docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +992,154 @@ eclagent:
+      create the LDAP "secret" similar to the following example. Note the
+      secret name, "admincredssecretname" used in this example. The LDAP
+      Administrator "username" and "password" key/values are required;
+      additional properties are allowed but ignored.</para>
+
+      <programlisting>kubectl create secret generic admincredssecretname --from-literal=username=hpcc_admin \
+  --from-literal=password=t0pS3cr3tP@ssw0rd
+
+kubectl get secret admincredssecretname
+</programlisting>
+
+      <para>This effectively sets the values for the Administrator's username
+      and password in Kubernetes.</para>
+
+      <para>For more information about Kubernetes see the appropriate
+      Kubernetes documentation for your implementation.</para>
+
+      <sect3 id="Deploying_TheKubernetesSecret">
+        <title>Deploying the Kubernetes Secret</title>
+
+        <para>Override the "secrets:" section in
+        HPCC-Platform/helm/hpcc/values.yaml, or your own customization
+        chart.</para>
+
+        <para>Create a unique key name used to reference the secret (this will
+        be the mounted file system name of the secret), and set it to the
+        secret value ("admincredssecretname") that you created in the
+        configuring section. In this example, the key named
+        "admincredsmountname," and optionally define an additional alternate
+        one "admincredsaltmountname" which could be used with another Active
+        Directory server, for instance. Note that the "admincredsmountname"
+        key/value pair already exists as a default in the HPCC Systems
+        delivered values.yaml file. The key is referenced in the component's
+        ldap.yaml file. You may override these and add additional key/values
+        as needed.</para>
+
+        <programlisting>  secrets:
+     authn:
+       admincredsmountname: "admincredssecretname"       #exernalize LDAP Admin creds
+       admincredsaltmountname: "admincredsaltsecretname" #exernalize alternate LDAP Admin creds
+</programlisting>
+      </sect3>
+
+      <sect3 id="ENABLE_k8S_LDAP">
+        <title>Enable LDAP Authentication</title>
+
+        <para>In the delivered
+        HPCC-Platform/esp/applications/common/ldap/ldap.yaml file, the
+        "ldapAdminSecretKey" is already set to the key mount name illustrated
+        in the example above. To enable the LDAP authentication and to
+        override this value, override the ESP/ECLWatch helm component located
+        in values.yaml :</para>
+
+        <programlisting>esp:
+- name: eclwatch
+  application: eclwatch
+  auth: ldap
+  ldap:
+    ldapAddress: "myldapserver"
+    ldapAdminSecretKey: "admincredsaltmountname"   # use alternate secrets creds
+</programlisting>
+
+        <para> </para>
+      </sect3>
+    </sect2>
+
+    <sect2 id="Configuring_For_HashoCorpVault">
+      <title>Configuring LDAP for HashiCorp Vault</title>
+
+      <para>To configure secrets for HashiCorp Vault, from it's command line
```

Comment:
Similar to above, something like, execute the following command with the command line and define the values inserted by the user with a Where:   xxxx is yyyyy,  aaaa is bbbbbb